cspell.json

```diff
@@ -61,7 +61,8 @@
     "Vendicated",
     "Webauthn",
     "weblate",
-    "XLARGE"
+    "XLARGE",
+    "splashrenderer"
```

Comment:
Use camelCase for the directory name, then you won't need this.

Reply:
👍 Will do, thanks for the info!